app/frontend/src/AppRoute.tsx

```diff
@@ -60,28 +56,34 @@ export default function AppRoute({
       render={({ location }) => (
         <>
           {isAuthenticated ? (
-            <Container
-              className={classNames({
-                [classes.nonFullScreenStyles]: variant !== "full-screen",
-                [classes.fullWidthContainer]: variant === "full-width",
-                [classes.fullscreenContainer]: variant === "full-screen",
-                [classes.standardContainer]: variant === "standard",
-              })}
-              maxWidth={
-                variant === "full-screen" || variant === "full-width"
-                  ? false
-                  : undefined
-              }
-            >
+            <>
               {isJailed ? (
                 <Redirect to={jailRoute} />
               ) : (
-                <>
-                  {variant !== "full-screen" && <Navigation />}
-                  <ErrorBoundary>{children}</ErrorBoundary>
-                </>
+                <>{variant !== "full-screen" && <Navigation />}</>
               )}
```

Comment:
I'm guessing this is necessary so the navigation bar isn't restricted by the container?

Also, you don't need this `Redirect` since there is already a redirect below, so you can remove it and do:
```typescript
{variant !== "full-screen" && <Navigation />}
```

Reply:
Yes, with `display:sticky`, the menu is fixed but also "occupies" its space, so it's no longer needed to have different paddings in order to fit the container. For that I had to put it out of the container. Now if the menu changes height, everything will adapt without further change.